core/src/main/java/hudson/PluginManager.java

```diff
@@ -2105,8 +2105,12 @@ public String toString() {
     }
     public static boolean FAST_LOOKUP = !SystemProperties.getBoolean(PluginManager.class.getName()+".noFastLookup");
 
-    public static final Permission UPLOAD_PLUGINS = new Permission(Jenkins.PERMISSIONS, "UploadPlugins", Messages._PluginManager_UploadPluginsPermission_Description(),Jenkins.ADMINISTER,PermissionScope.JENKINS);
-    public static final Permission CONFIGURE_UPDATECENTER = new Permission(Jenkins.PERMISSIONS, "ConfigureUpdateCenter", Messages._PluginManager_ConfigureUpdateCenterPermission_Description(),Jenkins.ADMINISTER,PermissionScope.JENKINS);
+    /** @deprecated use {@link Jenkins#ADMINISTER} instead */
```

Comment:
would love to see this as `@deprecated in Jenkins xxx use {@link Jenkins#ADMINISTER} instead`  (because we are not on JDK 9 or higher yet...